### ISSUE TITLE
custom learning rate schedule

### ISSUE THREAD

**Issue by JiajunBao** [flairNLP/flair#1174]
I want to first decay learning rate and then increase learning rate when the model is going to overfit. How could I use a custom learning rate scheduler into the trainer? I have a built-in scheduler here(https://github.com/zalandoresearch/flair/blob/5d62c8be1884beff3a8ee2208feb9b282e209cfe/flair/trainers/trainer.py#L194). I am not sure if I should pass a scheduler directly into the trainer and if there is a way to change the built-in one. 

Thank you.

**Comment by alanakbik** [comment]
Hi @JiajunBao the scheduler is currently fixed in the class so it cannot be changed. But we could add a PR to change this. What scheduler would you like to use?

**Comment by JiajunBao** [comment]
I want to first decay learning rate so that the model could learn more things. When the model starts to overfit, I want to increase learning rate back to see apparent overfitting. Normally, I add conditions to operate scheduler.step(). Since now the train function is wrapped in the trainer, I am not sure which one will work here. My current idea is to use `torch.optim.lr_scheduler.LambdaLR` to manually set the learning rate. Do you have any advice on which scheduler would work well here? Thank you!

**Comment by ciaochiaociao** [comment]
Any updates here?
